app/assets/javascripts/application.js.erb

```diff
@@ -21,7 +21,7 @@
 //= require_tree .
 
 BelAir = {
-  googleCalendarID: "<%= ENV['GOOGLE_CALENDAR_ID'] %>",
-  livestreamURL: "<%= ENV['LIVESTREAM_URL'] %>",
+  googleCalendarID: "<%= ENV['GOOGLE_CALENDAR_ID'].html_safe %>",
```

Comment:
Not sure why, but for some reason this fixed the issue that was causing the livestream not to work in the production app.